fabric-command-api-v1/src/main/java/net/fabricmc/fabric/api/client/command/v1/ArgumentBuilders.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.command.v1;
+
+import com.mojang.brigadier.arguments.ArgumentType;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.builder.RequiredArgumentBuilder;
+
+/**
+ * Provides helper methods for constructing argument builders
+ * for client-sided commands.
+ *
+ * <p>This is an alternative to the server-side helper methods in
+ * {@link net.minecraft.server.command.CommandManager}.
+ */
+public final class ArgumentBuilders {
```

Comment:
The class name should specify it is for client arguments.

Reply:
Renamed to `ClientArgumentBuilders`.